Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -636,8 +640,6 @@ public bool CreateCoinStake(List<StakeTx> stakeTxes, Block block, ChainedBlock c
             this.logger.LogTrace("Trying to reduce our staking UTXO set by adding additional inputs to coinstake transaction...");
             foreach (StakeTx stakeTx in setCoins)
             {
-                this.nodeLifetime.ApplicationStopping.ThrowIfCancellationRequested();
```

Comment:
Why was this removed from here?

Reply:
that was argued in #397, it is not needed here, this part of the code is quick and will complete in no time even for large miner, so i could either replace it with the construction as before or remove it, i couldn't find reason to keep it

Reply:
cool mate, thanks.